lib/passageidentity/user_api.rb

```diff
@@ -199,6 +199,8 @@ def create_v2(args: {})
     end
 
     def update_v2(user_id:, options: {})
+      raise ArgumentError, 'user_id is required.' unless user_id && !user_id.empty?
```

Comment:
We probably want options not to be empty too.

Reply:
oh yeah good point. added in [3f6fb30](https://github.com/passageidentity/passage-ruby/pull/116/commits/3f6fb304d8a8deffe27836e71822080e7e507870)